docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,123 @@
+
+4. Will you identify deprecated functionality such as passtickets? **[where will this functionality be identified? are passtickets really deprecated? first FAQ says no.]**
+
+   We plan to identify deprecated functionality. We will announce how this functionality will be identified in the near future.
+
+5. How do you intend to work with the the bypass scheme?
+
+   The bypass scheme remains and you can still claim conformance for services that only provide public endpoints that do not require authentication.
+
+6. Do the dependency changes mean that I need Java 17 to run Zowe V3?
+
+   Yes. Java 17 is required to run the API Mediation Layer in Zowe V3.
+
+### Zowe Explorer for Visual Studio Code
+
+1. Will the functionality to convert Zowe V1 profiles to team configuration remain available for Zowe V3?
+
+Yes. However, users will not be able to use Zowe V1 profiles to connect to services on the mainframe.
+
+2. Will the APIs that Zowe CLI provides for extenders remain the same?
+
+   In broad terms, yes. However, some of the functionality that was available in Zowe V2 has been changed (or removed). The changes are included in these [CLI](https://ibm.ent.box.com/s/vqu92d82b4wk0i6fupo8glbrxvufn4zw) and [Imperative](https://github.com/zowe/imperative/issues/970) lists of breaking changes.
+
+### Zowe Application Framework, ZSS
+
+1. Is it possible to run 32-bit plug-ins at the same time as 64-bit plug-ins with ZSS?
+
+   No. ZSS runs in either 32-bit or 64-bit mode, which means it can accommodate only one type of plug-in.
+
+2. Are **[what kind of?]** libraries updated to the current latest versions?
+
+   In Zowe V3, the Application Framework uses Angular 16 and Webpack 5, which are the latest current releases.**[are there other libraries that aren't up-to-date, and if so, should we list them here?]**
+
+3. What React versions are supported by Zowe Desktop?
+
+   It is possible for Desktop extensions to use different versions of React. While the Desktop is not built on React, it is still compatible with extensions that import React.
+
+4. Is the ZEN application running on Zowe Desktop or on z/OS?
+
+   ZEN runs on a laptop/desktop and connects to z/OS via standard methods.
+
+5. Is it possible to validate and change the `zowe.yaml` file and job definition within ZEN?
+
+   Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
+
+   You are guided through the SMP/E installation steps.
+
+7. What is the connection between Zowe Store and ZEN?
+
+   Zowe Store is separate from ZEN, but they do share **[some commands? all commands?]** `zwe` commands.
```

Comment:
@balhar-jakub - please respond

Reply:
Sean Grady knows the answer.

Reply:
I've never heard of the Zowe Store.
But ZEN shouldn't be called ZEN. It's the "Zowe Server Install Wizard".
ZEN is a codename, hence the confusion.

Reply:
Can we remove this question as it appears this "Zowe Store" is not a current option?  Or is it?

Reply:
I would remove the question

Reply:
I removed the question.